app/src/main/AndroidManifest.xml

```diff
@@ -152,9 +161,22 @@
         <category android:name="android.intent.category.DEFAULT"/>
         <category android:name="android.intent.category.BROWSABLE"/>
       </intent-filter>
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW"/>
```

Comment:
ParseDeepLinkActivity is for parsing all deep links. EventActivity expects an extra with the id. I'd prefer to have one exported activity that handles all the incoming intents...